frontend/constants/serverAddress.js

```diff
@@ -1,3 +1,3 @@
 export default {
-    address: '192.168.0.156'
+    address: '192.168.88.14'
```

Comment:
prośba o powrót do poprzedniej wersji tej zmiennej

Reply:
this was mine address that I committed on accident, so that's not necessary